src/env.cc

```diff
@@ -492,6 +492,12 @@ Environment::~Environment() {
   CHECK_EQ(base_object_count_, 0);
 }
 
+AllocatedBuffer Environment::AllocateManaged(size_t size, bool checked) {
+  char* data = checked ? Allocate(size) : AllocateUnchecked(size);
+  if (data == nullptr) size = 0;
+  return AllocatedBuffer(this, uv_buf_init(data, size));
+}
```

Comment:
It would be nice to keep this method inline, I think.

Reply:
Done. I actually moved the method off `Environment` and made it a static on `AllocatedBuffer`.